src/ziggurat/messaging/connection.clj

```diff
@@ -1,66 +1,2 @@
-(ns ziggurat.messaging.connection
```

Comment:
if we have deleted this entire file, why not move it as a separate namespace inside the rabbitmq wrapper package?
We are as it is trying to separate it out into separate namespaces, right?